pkg/controller/directvolumemigration/rsync.go

```diff
@@ -58,15 +61,18 @@ const (
 	RsyncAttemptLabel = "migration.openshift.io/rsync-attempt"
 )
 
-// ensureRsyncEndpoint ensures that a new Endpoint is created for Rsync Transfer
-func (t *Task) ensureRsyncEndpoint() error {
+// ensureRsyncEndpoints ensures that new Endpoints are created for Rsync and Blockrsync Transfers
+func (t *Task) ensureRsyncEndpoints() error {
 	destClient, err := t.getDestinationClient()
 	if err != nil {
 		return liberr.Wrap(err)
 	}
 
 	dvmLabels := t.buildDVMLabels()
 	dvmLabels["purpose"] = DirectVolumeMigrationRsync
+	blockdvmLabels := t.buildDVMLabels()
```

Comment:
looks like this is redundant? can re-use dvm labels as-is

Reply:
I need 2 copies, since I am modifying it differently for block. I could make a copy and add the `app` label.